dbt/include/athena/macros/materializations/models/table/table.sql

```diff
@@ -24,6 +25,8 @@
     {{ set_table_classification(target_relation) }}
   {% endif %}
 
+  {{ adapter.add_lf_tags_to_table(target_relation.schema, identifier, lf_tags) }}
```

Comment:
lovely, shall we had
```
{% if lf_tags is not none %}
   {{ adapter.add_lf_tags_to_table(target_relation.schema, identifier, lf_tags) }}
{% endif %}
```
Just a safe guard to avoid not necessary function calls.